feat: document permissions (#528)
BeccaLyria/discord-documentation: docs/manual-commands.md

- `leaderboard` - Generates an embed containing the top ten users (by points) in the server, and the level + rank of the user who called the command.
- `level` - This command sends an embed containing the message author's level and current experience points.
- `list` - This command returns embed(s) containing a list of the servers Becca is in, and the owner information for each server. This command is restricted to the bot owner only.

> - `permissioncheck` - This command is locked to the `Manage Server` permission. When you run this command, Becca will check if she has the required permissions in the channel the command is used in, as well as at the server level. Becca will report the permissions she checked, the results for each permission, and a notice whether things need fixed or not.

Reviewer: ```suggestion
- `permissioncheck` - This command is locked to the `Manage Server` permission. When you run this command, Becca will check if she has the required permissions in the channel the command is used in and at the server level. Becca will report the permissions she checked, the results for each permission, and a notice whether things need to be fixed or not.
```
